app/controllers/idv/capture_doc_controller.rb

```diff
@@ -14,6 +14,13 @@ class CaptureDocController < ApplicationController
       analytics_id: Analytics::DOC_AUTH,
     }.freeze
 
+    def return_to_sp
```

Comment:
do we need to explicitly add a `track_event` call to this action?

Reply:
The purpose of this is basically to set a default value for the `step` parameter before passing along to [`ReturnToSpController#failure_to_proof`](https://github.com/18F/identity-idp/blob/952172145629d556aef6fa1ebc46c0b93cfd2a81/app/controllers/return_to_sp_controller.rb#L10-L18), which does the actual logging.

Double-redirect is maybe not the greatest, but doesn't seem too concerning? Otherwise could try to find another way for the two to share some common logic, or just copy the `track_event` code in here.

Reply:
I mean if it was easy, would love to minimize # of redirects, but otherwise two redirects like this is not the end of the world

Reply:
Looked at it a bit. I think the next best option is basically copying the same logic from `ReturnToSpController`, so I'll just keep it as-is.